build/create_arm_debootstrap.sh

```diff
@@ -238,9 +238,8 @@ install -m 444 -o root -g root ${REP}/script/hypercube/install.html $TARGET_DIR/
 if [ $INSTALL_YUNOHOST ] ; then
   chroot_deb $TARGET_DIR "mkdir -p /run/systemd/system/"
 
-  chroot_deb $TARGET_DIR "$APT git"
-  chroot_deb $TARGET_DIR "git clone https://github.com/YunoHost/install_script /tmp/install_script"
-  chroot_deb $TARGET_DIR "cd /tmp/install_script && ./install_yunohost -a -d ${INSTALL_YUNOHOST_DIST}"
+  chroot_deb $TARGET_DIR "wget -O /tmp/install_yunohost install.yunohost.org/${DEBIAN_RELEASE} && chmod +x /tmp/install_yunohost"
```

Comment:
I suggest to replace install.yunohost.org by https://install.yunohost.org

Is a reason for not doing the request with a secure way ?

Reply:
rectifié, merci !